cloud/azure/modules/app/outputs.tf

```diff
@@ -2,8 +2,8 @@ output "app_service_default_hostname" {
   value = "https://${azurerm_app_service.civiform_app.default_site_hostname}"
 }
 
-output "pg_server_privatelink_fqdn" {
-  value = local.postgres_private_link
+output "app_service_canary_hostname" {
+  value = "https://${azurerm_app_service_slot.canary.default_site_hostname}"
 }
 
```

Comment:
do we need to add the hostname for the non canary?

Reply:
I think that's already in there!

Reply:
I took out the postgres link because it's useless outside the vnet and I didn't want to clutter the output